source/uv_interop.c

```diff
@@ -106,17 +106,23 @@ static void s_uv_closed(uv_handle_t *handle) {
     aws_mem_release(ctx->allocator, ctx);
 }
 
-void aws_uv_context_release(struct aws_uv_context *ctx) {
-    aws_mutex_lock(&ctx->command_queue.mutex);
-    /* For now, don't bother supporting a final flush, it shouldn't be necessary, as when refs are
-       dropped the owning object should be on its way to death */
+static void s_context_release(void *user_data) {
+    struct aws_uv_context *ctx = user_data;
+    /* this should be the last command in the queue, so the queue should be empty */
```

Comment:
Remember when I thought we wouldn't need this? Oops.

For now, we do. If we like this mechanism, ```napi_threadsafe_function``` is identical to this in functionality, so I can just transition away from this code and use the node-provided API. We didn't originally because we couldn't depend on NAPI > 1, but we have to now, as those versions of node are past LTS.